tools/cli/datalab.py

```diff
@@ -181,6 +181,9 @@ def gcloud_compute(
       stdin: The 'stdin' argument for the subprocess call
       stdout: The 'stdout' argument for the subprocess call
       stderr: The 'stderr' argument for the subprocess call
+      wait: Whether or not to wait for the command to complete
+    Returns:
+      A subprocess.Popen object iff `wait` is True
```

Comment:
This seems like it's backwards; the Popen object is returned iff wait is false. Am I misreading? Same below

Reply:
You're right; thanks for catching that.